scripts/lib/install.sh

```diff
@@ -0,0 +1,63 @@
+install_aws_lb_controller() {
```

Comment:
Is this not handled by the ginkgo tests?

Reply:
I don't think ginkgo installs it by default. It's similar to how we test using eksctl (e2e test)